src/components/panels/hours-panel-date-viewer.js

```diff
@@ -0,0 +1,586 @@
+import { Heading, Icon, Link, Margins, MEDIA_QUERIES, SPACING, TYPOGRAPHY } from '../../reusable';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useStateValue } from '../use-state';
+
+const dateFormat = (string, abbreviated = false) => {
+  if (abbreviated) {
+    return string.toLocaleString('en-US', {
+      day: 'numeric',
+      month: 'short'
+    });
+  }
+  return string.toLocaleString('en-US', {
+    day: 'numeric',
+    month: 'long',
+    weekday: 'long',
+    year: 'numeric'
+  });
+};
+
+const IconWrapper = (props) => {
+  return (
+    <span
+      css={{
+        display: 'inline-block',
+        marginTop: '-2px'
+      }}
+      {...props}
+    />
+  );
+};
```

Comment:
This component might not even be needed. They are only used in `PreviousNextWeekButton`, which is inside a `flex` element. You could add `align-items: center;` and `gap: SPACING['2XS']` to the parent, and remove the extra CSS used in the wrapped `Icon` components.